packages/edit-post/src/components/sidebar/post-schedule/style.scss

```diff
@@ -6,13 +6,15 @@
 
 	span {
 		display: block;
-		width: 45%;
-		flex-shrink: 0;
-		// Match padding on tertiary buttons for alignment.
-		padding: $grid-unit-15 * 0.5 0;
+		width: 30%;
+		margin-right: 8px;
```

Comment:
These changes can feel random but they are changes meant to unify the spacing and the proportions of these panels between site and post editors.

As I move more panels into the `editor` package, I'll try to use a dedicated UI component that I'll be sharing between all these panels to avoid recreating the same design over and over again.

Reply:
To be honest I'll just keep it as is for now as it's a bit random but I'm going to do better in the next PR I promise :P